tools/ctc_segmentation/scripts/prepare_data.py

```diff
@@ -140,6 +140,7 @@ def split_text(
         .replace("´", "")
         .replace("--", " -- ")
         .replace("-", " - ")
+        .replace(". . .", "...")
```

Comment:
Is this change supposed to be in this PR?

Reply:
thanks for catching it, will revert.